test/Libraries/PackageManagerTests/PackageLoaderCustomTest.cs

```diff
@@ -41,19 +41,27 @@ public void LoadFromMultipleLocationTest()
             Assert.AreEqual(loader.DefaultPackagesDirectory,
                 Path.Combine(TestDirectory, "pkgs", "multiple_locations", "folder1", "packages"));
 
-            Assert.AreEqual(2, loader.LocalPackages.Count());
+            var expectedLoadedPackageNum = 0;
+            foreach(var pkg in loader.LocalPackages)
+            {
+                if(pkg.Name == "Custom Rounding" || pkg.Name == "GetHighest")
+                {
+                    expectedLoadedPackageNum++;
+                }
+            }
+            Assert.AreEqual(2, expectedLoadedPackageNum);
 
-            var pkg = loader.LocalPackages.ElementAt(0);
+            var firstPkg = loader.LocalPackages.ElementAt(0);
```

Comment:
Can we be sure these packages will be loaded in the same no matter what other packages may be installed?
Wouldn't it be safer to search the package by name instead?

Reply:
Good idea. I will update this code